.github/workflows/iroha2-release.yml

```diff
@@ -31,8 +31,11 @@ jobs:
           registry: docker.soramitsu.co.jp
           username: ${{ secrets.HARBOR_USERNAME }}
           password: ${{ secrets.HARBOR_TOKEN }}
+      - name: Check changed files
+        uses: jitterbit/get-changed-files@v1
+        id: base_image
       - name: Build and push iroha2-base image
-        # TODO Add `if` condition #2732
+        if: contains(steps.base_image.outputs.added_modified, 'Dockerfile.base')
```

Comment:
#2732 can be closed, but in #2690 please consider the layer caching instead of having the base image

Reply:
[Failing test](https://github.com/hyperledger/iroha/actions/runs/3099709697/jobs/5019371879)

Reply:
I think this failed test is not related to the current PR changes.